### ISSUE TITLE
create stops at fixed (geo) locations

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#13816]
This is relevant for jupedsim where stops / waypoints could be anywhere in the plane. For cars / other models it should be mapped to the closest edge

**Comment by palvarezlopez** [comment]
@behrisch also in Netedit?

**Comment by behrisch** [comment]
> @behrisch also in Netedit?

Yes but it seems the JuPedSim people found an easier solution with edges so it is not urgent
